Error for solutions using `import`

When I publish the following code:

```go
// Package twofer allows on to generate a sharing message.
package twofer

import "fmt"

// ShareWith takes a name and returns a sharing greeting.
func ShareWith(name string) string {
	if name == "" {
		name = "you"
	}

	return fmt.Sprintf("One for %s, one for me.", name)
} 
```

there is an error creating a representation for it:

```shell
2023/09/21 12:17:55 Creating representation for `two-fer` exercise in folder `/mnt/exercism-iteration/`
2023/09/21 12:17:56 /mnt/exercism-iteration/two_fer.go:4:8: could not import fmt (can't find import: "fmt")
github.com/exercism/go-representer/representation.(*Representation).parseInfo
        /go-representer/representation/representation.go:90
github.com/exercism/go-representer/representation.(*Representation).ParseAST
        /go-representer/representation/representation.go:67
github.com/exercism/go-representer.Extract
        /go-representer/representer.go:13
main.run
        /go-representer/cmd/representer/main.go:60
main.main
        /go-representer/cmd/representer/main.go:50
runtime.main
        /usr/local/go/src/runtime/proc.go:250
runtime.goexit
        /usr/local/go/src/runtime/asm_amd64.s:1598
failed to parse solution AST
```

--- a/.github/workflows/test.yml
+++ b/.github/workflows/test.yml
@@ -38,7 +38,7 @@ jobs:
           context: .
           push: false
           load: true
-          tags: exercism/go-test-runner
+          tags: exercism/go-representer
           cache-from: type=gha
           cache-to: type=gha,mode=max
 

--- a/Dockerfile
+++ b/Dockerfile
@@ -3,32 +3,26 @@ FROM golang:1.20.1-alpine3.17 as builder
 # Install SSL ca certificates
 RUN apk update && apk add git && apk add ca-certificates
 
-# Create appuser
-RUN adduser -D -g '' appuser && mkdir /go-representer
+# Add a non-root user to run our code as
+RUN adduser --disabled-password appuser
 
-# source code
-WORKDIR /go-representer
-COPY ./go.mod /go-representer/go.mod
+# Copy the source code into the container
+# and make sure appuser owns all of it
+COPY --chown=appuser:appuser . /opt/representer
 
-# download dependencies
+# Build and run the representer with appuser
+USER appuser
+
+# This populates the build cache with the standard library
+# so future compilations are faster
+RUN go build std
+
+WORKDIR /opt/representer
+
+# Download dependencies
 RUN go mod download
 
-# get the rest of the source code
-COPY . /go-representer
-
 # build
-RUN go generate .
-RUN GOOS=linux GOARCH=amd64 go build --tags=build -o /go/bin/representer ./cmd/representer
+RUN go build --tags=build -o /opt/representer/bin/representer ./cmd/representer
 
-# Build a minimal and secured container
-# The ast parser needs Go installed for import statements.
-# Therefore, unfortunately we cannot build from scratch as we would normally do with Go.
-FROM golang:1.20.1-alpine3.17
-COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
-COPY --from=builder /etc/passwd /etc/passwd
-COPY --from=builder /go/bin /opt/representer/bin
-COPY bin/run.sh /opt/representer/bin/
-
-USER appuser
-WORKDIR /opt/representer
-ENTRYPOINT ["/opt/representer/bin/run.sh"]
+ENTRYPOINT ["sh", "/opt/representer/bin/run.sh"]

--- a/bin/run-tests.sh
+++ b/bin/run-tests.sh
@@ -18,11 +18,14 @@ filenames="representation.txt mapping.json"
 for test_dir in testdata/*/*; do
     test_dir_name=$(basename "${test_dir}")
     test_dir_path=$(realpath "${test_dir}")
+    output_dir_path="/tmp/${test_dir_name}"
 
-    bin/run.sh "${test_dir_name}" "${test_dir_path}" "${test_dir_path}"
+    mkdir -p "${output_dir_path}"
+
+    bin/run.sh "${test_dir_name}" "${test_dir_path}" "${output_dir_path}"
 
     for filename in $filenames; do
-        actual_filepath="${test_dir_path}/${filename}"
+        actual_filepath="${output_dir_path}/${filename}"
         expected_filepath="${test_dir_path}/expected_${filename}"
         
         echo "${test_dir_name}: comparing ${filename} to expected_${filename}"

